elasticsearch/src/main/java/io/micronaut/elasticsearch/DefaultElasticsearchClientFactory.java

```diff
@@ -54,6 +60,18 @@ RestClient restClient(DefaultElasticsearchConfigurationProperties elasticsearchC
         return restClientBuilder(elasticsearchConfiguration).build();
     }
 
+    /**
+     * @param elasticsearchConfiguration The {@link DefaultElasticsearchConfigurationProperties} object
+     * @return The Elasticsearch Client
+     */
+    @Bean
```

Comment:
Do we need a `preDestroy` here?

Reply:
There is no similar method on the client that I could find. `shutdown` shuts down the cluster node - not the best idea I guess.

Reply:
Or rather returns an object that can do that - anyway there is no `close` method.